caravel/assets/javascripts/explorev2/components/NotGroupBy.jsx

```diff
@@ -19,57 +19,43 @@ const defaultProps = {
   orderings: [],
 };
 
-class NotGroupBy extends React.Component {
-  changeColumns(columns) {
-    this.props.actions.setNotGroupByColumns(columns);
-  }
-  changeOrderings(orderings) {
-    this.props.actions.setOrderings(orderings);
-  }
-  render() {
-    return (
-      <div className="panel space-1">
-        <div className="panel-header">Not GroupBy</div>
-        <div className="panel-body">
-          <div className="row">
-            <h5 className="section-heading">Columns</h5>
-            <Select
-              multi
-              name="select-column"
-              placeholder="Select columns"
-              options={this.props.columnOpts}
-              value={this.props.columns}
-              autosize={false}
-              onChange={this.changeColumns.bind(this)}
-            />
-          </div>
-          <div className="row">
-            <h5 className="section-heading">Orderings</h5>
-            <Select
-              multi
-              name="select-orderings"
-              placeholder="Select orderings"
-              options={this.props.orderingOpts}
-              value={this.props.orderings}
-              autosize={false}
-              onChange={this.changeOrderings.bind(this)}
-            />
-          </div>
-        </div>
+const NotGroupBy = (props) => {
+  const selects = [];
+  selects.push({
+    key: 'columns',
+    title: 'Columns',
+    options: props.columnOpts,
+    value: props.columns,
+    multi: true,
+    width: '12',
+  });
+  selects.push({
+    key: 'orderings',
+    title: 'Orderings',
+    options: props.orderingOpts,
+    value: props.orderings,
+    multi: true,
```

Comment:
since most of the selects use the same `multi` and `width` values, could we add them as default values in the `SelectArray` component?